test/sequential/rolldown.withdraw.preOperation.test.ts

```diff
@@ -0,0 +1,578 @@
+          idFromNumber,
+          idToNumber,
+          sequencer,
+          "Ethereum",
+        ),
+      ),
+      Sudo.sudo(
+        await Rolldown.createForceManualBatch(
+          idFromNumber,
+          idToNumber,
+          sequencer,
+          "Ethereum",
+        ),
+      ),
+    );
+    const batchIdAfter = await getLastBatchId();
+    expect(batchIdAfter).toEqual(batchIdBefore + 2);
+  });
+
+  test("Given a manually batch generation, WHEN is not the forced, THEN start-end will be calculated automatically", async () => {
+    const events = await Sudo.batchAsSudoFinalized(
+      ...(await Rolldown.createABatchWithWithdrawals(
+        testUser,
+        gaspIdL1Asset.ethereum,
+        2,
+      )),
+      await Rolldown.createManualBatch("EthAnvil"),
+    );
+    const filteredEvent = JSON.parse(
+      JSON.stringify(
+        events.filter((event) => event.method === "TxBatchCreated"),
+      ),
+    );
+    expect(filteredEvent[0].event.data[4][0]).toEqual(nextRequestIdEth);
+    expect(filteredEvent[0].event.data[4][1]).toEqual(nextRequestIdEth + 1);
+  });
+
+  test("Given a manually batch generation, WHEN is the forced ( sudo ) , THEN start-end must be set manually", async () => {
+    const sequencer = JSON.parse(
+      JSON.stringify(await SequencerStaking.activeSequencers()),
+    ).Ethereum[0];
+    const events = await Sudo.batchAsSudoFinalized(
+      ...(await Rolldown.createABatchWithWithdrawals(
+        testUser,
+        gaspIdL1Asset.ethereum,
+        2,
+      )),
+      Sudo.sudo(
+        await Rolldown.createForceManualBatch(
+          nextRequestIdEth,
+          nextRequestIdEth + 1,
+          sequencer,
+          "Ethereum",
+        ),
+      ),
+    );
+    const filteredEvent = JSON.parse(
+      JSON.stringify(
+        events.filter((event) => event.method === "TxBatchCreated"),
+      ),
+    );
+    expect(filteredEvent[0].event.data[4][0]).toEqual(nextRequestIdEth);
+    expect(filteredEvent[0].event.data[4][1]).toEqual(nextRequestIdEth + 1);
+  });
+
+  test("Given a non-forced manually generation, WHEN triggered, start will be the latest index from the last generated MAX( manual or automatically ) and last will be the latest requestId", async () => {
+    await Sudo.batchAsSudoFinalized(
+      ...(await Rolldown.createABatchWithWithdrawals(
+        testUser,
+        gaspIdL1Asset.ethereum,
+        2,
+      )),
+    );
+    await signTx(
+      getApi(),
+      await Rolldown.createManualBatch("EthAnvil"),
+      testUser.keyRingPair,
+    ).then((events) => {
+      const res = getEventResultFromMangataTx(events);
+      expect(res.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+    });
+    const event = await Rolldown.waitForNextBatchCreated(
+      "Ethereum",
+      waitingBatchPeriod,
+    );
+    expect(event.range.from.toNumber()).toEqual(nextRequestIdEth);
+    expect(event.range.to.toNumber()).toEqual(nextRequestIdEth + 1);
```

Comment:
I think this is duplicated from old tests, right? Perhaps we can remove it.

Reply:
No, I want to check date from event. Removed